string MID
uint32 T_A0
uint32 T_A1
uint32 T_A2
uint32 T_A3
uint32 RangeId
uint64 TimeId
